tools/custom/raw-bytecode-analyser/README.md

```diff
@@ -0,0 +1,75 @@
+# Hedera Smart Contract Bytecode Analyzer
+
+This Python script analyzes the bytecode of Smart Contracts on the Hedera network.
```

Comment:
I would add something more specific, like what kind of detections are performed.

Reply:
I've extended the introduction. There are also more detailed information in the features section.